packs/core-abilities/weather-control.json

```
@@ -13,8 +13,8 @@
         "slug": "weather-control",
         "type": "generic",
         "cost": {
-          "activation": "simple",
-          "powerPoints": 6
+          "activation": "complex",
+          "powerPoints": 8
```

```suggestion
          "powerPoints": 6
```
see: https://ptb.discord.com/channels/748601513835888682/1176183212955942922/1351852469348601910